README.md

```diff
@@ -90,155 +92,128 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Fixed issue causing zoning not to happen until the cursor is moved.
- - Improved monitor identification logic to mitigate issues causing layout reset.
- - Fixed issue where default layout was applied instead of blank layout.
+ - Fixed an issue where FancyZones wouldn't register a change to the "Switch between windows in the current zone" setting.
+ - Added a Setting to enable the behavior of clicking the middle mouse button to toggle multiple zone spanning.
 
 ### File Locksmith
 
- - Added setting to show only in extended context menu.
+ - Fixed a File Explorer crash when deleting a file, upgrading PowerToys and then trying to right-click the background of a folder in File Explorer.
+ - UI tweaks. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
 
 ### File Explorer add-ons
 
- - Developer files preview support for .vsconfig, .sln, .vcproj, .vbproj, .fsproj and .vcxproj files. (This was a hotfix for 0.70)
- - Developer files preview support .vbs, .inf, .gitconfig, .gitattributes and .editorconfig files. (This was a hotfix for 0.70) Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Changed order of developer files preview` context menu items. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Developer files preview support for .gitignore files. (This was a hotfix for 0.70) Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Fixed issue causing preview pane flickering on file selection and resizing. Thanks [@tanchekwei](https://github.com/tanchekwei)!
+ - Updated the Monaco dependency for Developer Files Preview, supporting new files extensions and fixing issues. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
 
-### Hosts
+### Hosts File Editor
 
- - Improved UX by adding keyboard shortcuts. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Added setting to select the file encoding. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed parsing of commented lines with an address and host in the middle of the comment. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed issue on adding first entry and improve empty hosts list UI. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Added logic to handle more than 9 hosts per entry (Windows limitation) by splitting them into separate entries. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Consolidated the way the Hosts application is launched. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - UI tweaks. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
 
-### ImageResizer
+### Installer
 
- - Added Enter key event handler when setting width/height of the new custom size.
+ - Refactored the Monaco dependency inclusion. What to install is now being generated automatically. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
```

Comment:
```suggestion
 - Refactored the Monaco dependency inclusion. What to install is now being generated automatically.
```
This was Stefan :)

Reply:
I've checked with Stefan, you initiated it and also did a significant part, so it's OK to attribute you :)
Thanks for the review here, by the way.

Reply:
Ah, ok. I've gone deeper with Stefan. It seems the installer part ended up being him. Alright :) Thanks for the notice.